### DISCUSSION TITLE
Formatic with Full Static Caching

### DISCUSSION THREAD

**Discussion by lakkes-ra** [lucky-media/formatic#11] [Q&A] [answered]
Hi 👋, 
we used Formatic for a customer project and integrated it into an existing project. We saw inside the project files that the formatic forms get postet with JavaScript, but we couldn't use it with full static caching, since it's to depending on the Laravel XSRF Token from the cookies, that in some cases doesn't exist.

Since we implemented on our own, it's our problem, but I wanted to ask if formatic should natively work with full static caching. Maybe we just used it wrong or we have to extend it ourselfes with catching a dynamic token with JS. 

Kind regards,
Lukas

**Comment by lokmanm** [comment]
Hey @lakkes-ra.

Yeah, we can add support to Formatic so its all frontend and we can grab the CSRF token with an API Call. 

I will leave this issue open until we add support for this.

Thanks!

**Reply by lakkes-ra** [reply]
Awesome, thank you!

**Comment by lokmanm** [comment, marked as answer]
Closing this as we have released this a while back.
